tests/sysversioninfo.pyi

```diff
@@ -29,3 +29,21 @@ if sys.version_info <= (3, 0): ...  # Y006 Use only < and >= for version compari
 if sys.version_info < (3, 5): ...
 if sys.version_info >= (3, 5): ...
 if (2, 7) <= sys.version_info < (3, 5): ...  # Y002 If test must be a simple comparison against sys.platform or sys.version_info
+
+
+if sys.version_info >= (3, 10):
+    pass
+else:
+    pass
+
+if sys.version_info < (3, 10):  # Y066 When using if/else with sys.version_info, put the code for new Python versions first, e.g. "if sys.version_info >= (3, 10)"
+    pass
+else:
+    pass
+
+if sys.version_info >= (3, 10):
+    pass
+elif sys.version_info < (3, 8):  # Y066 When using if/else with sys.version_info, put the code for new Python versions first, e.g. "if sys.version_info >= (3, 8)"
+    pass
+else:
+    pass
```

Comment:
I'm being _very_ nitpicky here, but maybe we could make these tests slightly more "realistic". Usually the reason why we have this kind of patten in stubs is because e.g. a new Python version adds a new parameter to the function. So maybe we could have something like that here, to make it more obvious what the kind of pattern we're trying to test for is? Something like

```py
if sys.version_info >= (3, 10):
    def foo(x, *, bar=True, baz=False): ...
elif sys.version_info >= (3, 9):
    def foo(x, *, bar=True): ...
else:
    def foo(x): ...

Reply:
I'd call it thoroughness rather than being nitpicky - and it's always very much appreciated :)